Makefile

```diff
@@ -105,12 +105,21 @@ test: ## Run tests
 test-integration: ## Run integration tests
 	go test -v -tags=integration ./test/integration/...
 
-.PHONY: test-e2e
-test-e2e: ## Run e2e tests
-	PULL_POLICY=IfNotPresent $(MAKE) docker-build
+.PHONY: full-capd-e2e
```

Comment:
Just a suggestion but maybe it might be nice to prefix them with `test`.
E.g. `make test-capd-e2e-full`